### PR TITLE
feat: Implement service pricing component

### PR DESCRIPTION
**Pull request by Trishu-Patel**
image
image

### CODE REVIEW

**Review by lnicely — CHANGES_REQUESTED** [review]

File: packages/ember/src/components/mktg/service-pricing.gts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        class="col-auto fs-5
+          {{if
+            @descriptionDisabled
+            'fw-normal text-decoration-underline text-light-emphasis'
+          }}"
+      >
+        {{@description}}
+      </div>
+    </div>
+    {{yield (component Addon)}}
```
Comment (ChesneyJulian) [inline packages/ember/src/components/mktg/service-pricing.gts:74]: Should we make this a yield hash so the implementation is more aligned with other components of this nature?

Comment (ChesneyJulian) [reply packages/ember/src/components/mktg/service-pricing.gts:74]: Implementation in the test app would look more like this:
`<Mktg::ServicePricing as |Service|>
    <Service.Addon />
</Mktg::ServicePricing>`

File: packages/ember/src/components/mktg/service-pricing.gts [hunk 2]
```diff
@@ -0,0 +1,78 @@
+import type { NrgIconValue } from '../../icon-types';
+import type { TOC } from '@ember/component/template-only';
+import { concat } from '@ember/helper';
+import type { ComponentLike } from '@glint/template';
+
+interface AddonSignature {
+  Args: {
+    addon?: string;
```
Comment (lnicely) [inline packages/ember/src/components/mktg/service-pricing.gts:8]: Can we rename this to `label`?

File: packages/ember-test-app/app/templates/marketing.hbs [hunk 3]
```diff
@@ -198,5 +198,51 @@
...
+      @serviceName="TV"
+      @description="Add (optional)"
+      @icon="bi-tv"
+      @descriptionDisabled = {{true}}
+    />
+    <Mktg::ServicePricing
+      @serviceName="Phone"
+      @description="Add (optional)"
+      @icon="bi-telephone"
+      @descriptionDisabled = {{true}}
```
Comment (ChesneyJulian) [inline packages/ember-test-app/app/templates/marketing.hbs:245]: I think we collectively decided to use the @active param for determining the "Select", if not active and no selection made, then gray and underlined, if not active and selection is made, bolded price

File: packages/ember-test-app/app/templates/marketing.hbs [hunk 4]
```diff
@@ -198,5 +198,51 @@
         </div>
       </div>
     </form>
+
+    <Mktg::ServicePricing
+      @serviceName="Fiber"
+      @serviceType="The Gig"
```
Comment (lnicely) [inline packages/ember-test-app/app/templates/marketing.hbs:204]:
```suggestion
      @name="Fiber"
      @type="The Gig"
```

File: packages/ember-test-app/app/templates/marketing.hbs [hunk 5]
```diff
@@ -198,5 +198,60 @@
         </div>
       </div>
     </form>
+
+    <Mktg::ServicePricing
+      @label="Fiber"
+      @type="The Gig"
```
Comment (lnicely) [inline packages/ember-test-app/app/templates/marketing.hbs:204]: I don't see the type property listed in the component. Should this be `package`?

**Review by lnicely — APPROVED** [review]

**Pull request merged**